room_cancelation/auto_cancel_event.py

```diff
@@ -0,0 +1,57 @@
+from api.room.models import Room as RoomModel
+from api.events.models import Events as EventsModel
+from datetime import datetime
+from helpers.calendar.credentials import Credentials
+from dateutil.relativedelta import relativedelta
+from helpers.calendar.events import RoomSchedules
+
+
+class UpdateRecuringEvent():
+    def update(self):
+        query = RoomModel.query
+        now = datetime.utcnow().isoformat()+"Z"
+        next_day = (datetime.utcnow() + relativedelta(day=1)).isoformat()+"Z"
```

Comment:
@C3real-kill3r is this meant to be `relativedelta` or `relativedate`?

Reply:
Relative delta is an inbuilt subclass of the dateutil class that can perform variou functions relating to timedifference. [This](https://dateutil.readthedocs.io/en/stable/relativedelta.html) should clarify that for you. I hope that was clear.

Reply:
@kampkelly for `relativedate`  we'd use the syntax `RelativeDate(date, n)` where n is the number of days plus or minus the date, this would only limit us to expressions such as `2019-02-06` and yet in our case we are manipulating much more than that... Aslo the date format that the calendar API accepts is much more different and this was the most robust way to handle it.

Reply:
Okay, it is clear now.